<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QConsolePropPage</class>
 <widget class="QWidget" name="QConsolePropPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>250</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>550</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Console</string>
  </property>
  <widget class="QGroupBox" name="groupBoxMessage">
   <property name="geometry">
    <rect>
     <x>282</x>
     <y>116</y>
     <width>141</width>
     <height>123</height>
    </rect>
   </property>
   <property name="title">
    <string>Message params</string>
   </property>
   <widget class="QToolButton" name="toolButtonBkgColorS">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelBkgColorS">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>20</y>
      <width>87</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Background color</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonTextColor">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>44</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelTextColor">
    <property name="geometry">
     <rect>
      <x>38</x>
      <y>44</y>
      <width>65</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Text color</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonFont">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>66</y>
      <width>103</width>
      <height>47</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>170</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Light">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Midlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>212</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Dark">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>85</red>
          <green>127</green>
          <blue>127</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Mid">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>113</red>
          <green>170</green>
          <blue>170</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="BrightText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>170</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Shadow">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="AlternateBase">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>212</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ToolTipBase">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>220</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ToolTipText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>170</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Light">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Midlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>212</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Dark">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>85</red>
          <green>127</green>
          <blue>127</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Mid">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>113</red>
          <green>170</green>
          <blue>170</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="BrightText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>170</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Shadow">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="AlternateBase">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>212</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ToolTipBase">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>220</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ToolTipText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>85</red>
          <green>127</green>
          <blue>127</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>170</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Light">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Midlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>212</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Dark">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>85</red>
          <green>127</green>
          <blue>127</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Mid">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>113</red>
          <green>170</green>
          <blue>170</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>85</red>
          <green>127</green>
          <blue>127</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="BrightText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>85</red>
          <green>127</green>
          <blue>127</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>170</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>170</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Shadow">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="AlternateBase">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>170</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ToolTipBase">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>220</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ToolTipText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Font sample</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QTableWidget" name="tableWidgetStyles">
   <property name="geometry">
    <rect>
     <x>434</x>
     <y>46</y>
     <width>107</width>
     <height>195</height>
    </rect>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>2</number>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>55</number>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>20</number>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>20</number>
   </attribute>
   <column>
    <property name="text">
     <string>Style Id</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Enable</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="labelStyles">
   <property name="geometry">
    <rect>
     <x>438</x>
     <y>8</y>
     <width>100</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Styles list</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelStyles1">
   <property name="geometry">
    <rect>
     <x>438</x>
     <y>24</y>
     <width>100</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>right click to manage</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButtonBkgColor">
   <property name="geometry">
    <rect>
     <x>142</x>
     <y>220</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelBkgColor">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>117</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Panel Background color</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxSpasing">
   <property name="geometry">
    <rect>
     <x>232</x>
     <y>216</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>40</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelSpasing">
   <property name="geometry">
    <rect>
     <x>182</x>
     <y>220</y>
     <width>45</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Spasing</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxLog">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>92</y>
     <width>89</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Enable loging</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxFileCount">
   <property name="geometry">
    <rect>
     <x>268</x>
     <y>52</y>
     <width>47</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>999999</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelFileName">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>56</y>
     <width>97</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>File name templete</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditFileName">
   <property name="geometry">
    <rect>
     <x>108</x>
     <y>52</y>
     <width>153</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditFileDir">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>16</y>
     <width>301</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelFileDir">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Log to directory</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButtonDir">
   <property name="geometry">
    <rect>
     <x>398</x>
     <y>16</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="QConsole.qrc">
     <normaloff>:/Resources/standardbutton-save-32.png</normaloff>:/Resources/standardbutton-save-32.png</iconset>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="QConsole.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>toolButtonBkgColor</sender>
   <signal>clicked()</signal>
   <receiver>QConsolePropPage</receiver>
   <slot>OnButtonBkgColorClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>413</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>424</x>
     <y>14</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonBkgColorS</sender>
   <signal>clicked()</signal>
   <receiver>QConsolePropPage</receiver>
   <slot>OnButtonColorClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>400</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>427</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonTextColor</sender>
   <signal>clicked()</signal>
   <receiver>QConsolePropPage</receiver>
   <slot>OnButtonColorClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>403</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>429</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonFont</sender>
   <signal>clicked()</signal>
   <receiver>QConsolePropPage</receiver>
   <slot>OnButtonFontClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>393</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>429</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableWidgetStyles</sender>
   <signal>customContextMenuRequested(QPoint)</signal>
   <receiver>QConsolePropPage</receiver>
   <slot>OnCustomContextMenuRequest(QPoint)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>456</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>371</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableWidgetStyles</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>QConsolePropPage</receiver>
   <slot>OnStyleSelectionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>481</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableWidgetStyles</sender>
   <signal>itemChanged(QTableWidgetItem*)</signal>
   <receiver>QConsolePropPage</receiver>
   <slot>OnStyleIdChage(QTableWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>487</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonDir</sender>
   <signal>clicked()</signal>
   <receiver>QConsolePropPage</receiver>
   <slot>OnDirectoryButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>356</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>358</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnButtonColorClicked()</slot>
  <slot>OnButtonFontClicked()</slot>
  <slot>OnButtonBkgColorClicked()</slot>
  <slot>OnCustomContextMenuRequest(QPoint)</slot>
  <slot>OnStyleSelectionChanged()</slot>
  <slot>OnStyleIdChage(QTableWidgetItem*)</slot>
  <slot>OnDirectoryButtonClicked()</slot>
 </slots>
</ui>
